src/config/index.ts

```diff
@@ -40,7 +40,7 @@ const envVarsSchema = Joi.object()
         SESSION_SECRET: Joi.string().required().description('Session secret'),
         REDIS_HOST: Joi.string().required().description('Redis host'),
         REDIS_PORT: Joi.string().required().description('Redis port'),
-        REDIS_PASSWORD: Joi.string().required().description('Redis password'),
+        REDIS_PASSWORD: Joi.string().required().description('Reids password').allow(''),
```

Comment:
@Behzad-rabiei We should not allow `REDIS_PASSWORD` to be empty?  🤔

Reply:
@scientiststwin Redis server can be use without password.

Reply:
Redis enters a special mode called protected mode when it is executed with the default configuration (binding all the interfaces) and without any password in order to access it.